.github/workflows/linux.yml

```diff
@@ -68,13 +68,10 @@ jobs:
 
     # Allow skipped Lint
     # Let Lint fail on pull requests
-    if: |
-      ${{
-        success()
-        || needs.Lint.result == 'skipped'
-        || ( needs.Lint.result == 'failure'
-             && github.event_name == 'pull_request' )
-      }}
+    if: ${{ !cancelled() && (success() ||
+           needs.Lint.result == 'skipped' ||
+           (needs.Lint.result == 'failure' && github.event_name == 'pull_request')
+          )}}
```

Comment:
Tiny nitpick, but this indentation looks unaligned.

Reply:
Done.